gammapy/background/off_data_background_maker.py

```diff
@@ -203,3 +224,81 @@ def save_models(self, modeltype):
         """
         for ngroup in range(self.ntot_group):
             self.save_model(modeltype, ngroup)
+
+    def make_bkg_model_link(self, data_store, modeltype, out_dir_background_model=None, filename_obs_group_table=None):
+        """
+
+        Parameters
+        ----------
+        data_store : `~gammapy.data.DataStore`
+            Create a `~gammapy.data.DataStore` for the runs for which ones we want to compute a background model
+        modeltype : {'3D', '2D'}
+            Type of the background modelisation
+        out_dir_background_model:  str
+            directory where are located the backgrounds models for each band in zenith and efficiency
+        filename_obs_group_table : str
+            name of the file containing the `~astropy.table.Table` with the group infos
+
+
+        Returns
+        -------
+        index_table_bkg : `~astropy.table.Table`
+            Index hdu table only for the background in order to associate a bkg model for each observation
+        """
+
+        obs_table = data_store.obs_table
+        if not filename_obs_group_table:
+            filename_obs_group_table = self.group_table_filename
+        if not out_dir_background_model:
+            out_dir_background_model = data_store.hdu_table.meta["BASE_DIR"]
+        table_group = Table.read(filename_obs_group_table, format='ascii.ecsv')
+        axes = ObservationGroups.table_to_axes(table_group)
+        groups = ObservationGroups(axes)
+        obs_table = ObservationTable(obs_table)
+        obs_table = groups.apply(obs_table)
+
+        data = []
+        for obs in obs_table:
+            try:
+                group_id = obs['GROUP_ID']
+            except IndexError:
+                print('Found no GROUP_ID for {}'.format(obs["OBS_ID"]))
+                continue
+            row = dict()
+            row["OBS_ID"] = obs["OBS_ID"]
+            row["HDU_TYPE"] = "bkg"
+            row["FILE_DIR"] = str(out_dir_background_model)
+            row["FILE_NAME"] = self.filename(modeltype, group_id)
+            if modeltype == "2D":
+                row["HDU_NAME"] = "bkg_2d"
+                row["HDU_CLASS"] = "bkg_2d"
```

Comment:
In the docstring you claim to handle the `modeltype = "3D"` case as well.
I think it's easy to do this here, no?
Just add this?
```python
elif modeltype == '3D':
    row["HDU_NAME"] = "bkg_3d"
    row["HDU_CLASS"] = "bkg_3d"
else:
    raise ValueError('Invalid modeltype: {}'.format(modeltype))
```